conf/application.conf

```diff
@@ -36,12 +52,26 @@ play.http.router = prod.Routes
 # The below controllers are the default exceptions to this rule.
 
 controllers {
-  uk.gov.hmrc.pillar2externalteststub.controllers.MicroserviceHelloWorldController = {
+  # 300 is the default, you may need to change this according to your needs
```

Comment:
do we need this? If we do, it should only be 50

Reply:
I'll set to 50 as you suggest , cause if we take it all out it will result to default values